### PR TITLE
Add Prism.languages.shell to bash file as an alias

### LINKED ISSUE #1320: Add support for shell
Hi, could support for the shell language be added please?

### PR DESCRIPTION
**Pull request by paladox**
Fixes #1320

### CODE REVIEW

File: components/prism-bash.js [hunk 1]
```diff
@@ -70,6 +70,8 @@
 		'operator': /&&?|\|\|?|==?|!=?|<<<?|>>|<=?|>=?|=~/,
 		'punctuation': /\$?\(\(?|\)\)?|\.\.|[{}[\];]/
 	};
+	
+	Prism.languages.shell = Prism.languages.bash;
```
Comment (Golmote) [inline components/prism-bash.js]: Could you move this down at the very bottom, just before `})(Prism);` please?

Comment (paladox) [reply components/prism-bash.js]: Yep, thanks, done.

**Comment by Golmote** [comment]
Great! Thanks for contributing!

**Comment by paladox** [comment]
@Golmote thank you :).

Im trying to switch from highlight.js to prism in gerrit https://gerrit-review.googlesource.com/c/gerrit/+/163090 :)

**Pull request merged**